### ISSUE TITLE
[QUESTION] Version 2.0.0rc5 doesn't have delete methods for entities?

### ISSUE THREAD

**Issue by cabjr** [milvus-io/pymilvus#694]
Hi guys, is it correct or I didn't found the methods to delete entities from the latest version of Milvus?

I remember that I used it in the previous versions (like 1.x), was it removed? if so, how are we supposed to remove entities from a collection?

Thanks in advance for your attention.

**Comment by XuanYang-cn** [comment]
Yes, delete will be supported on the official 2.0.0 version of Milvus. Now we are in 2.0.0rc5, this week we will release 2.0.0rc6.

Check this roadmap: https://milvus.io/docs/v2.0.0/roadmap.md#DML

BTW, if you're interested in How we're gonna implement delete. We're going to hold a tech meeting tomorrow (9.8) online https://wiki.lfaidata.foundation/pages/viewpage.action?pageId=43287098. 

Implementing delete is one of the 3 topics of this meeting. Looking forward to your attendance.
